export const INTERNAL_ERROR =
    "Ocorreu um erro interno, tente novamente mais tarde."
export const NOT_FOUND_ERROR = (resourceName: string) =>
    `Não foi possível encontrar o recurso "${resourceName}", ou ele não existe.`
